actor/src/main/scala/org/apache/pekko/dispatch/ThreadPoolBuilder.scala

```diff
@@ -173,19 +173,23 @@ object MonitorableThreadFactory {
   val doNothing: Thread.UncaughtExceptionHandler =
     new Thread.UncaughtExceptionHandler() { def uncaughtException(thread: Thread, cause: Throwable) = () }
 
+  private[pekko] class PekkoManagedBlocker[T](thunk: => T)
```

Comment:
why not just private?

Reply:
`private[pekko] class PekkoForkJoinWorkerThread` is private[pekko] too, maybe its useful somewhere